src/components/ActionModal.jsx

```diff
@@ -33,10 +33,12 @@ export const ActionModal = ({
       setLoading(true);
       try {
         if (id) {
+          console.log({ id });
```

Comment:
This `getMovie` method should be replaced with your mock API call.

Reply:
I'm not mocking anymore.  This is the way i've seen they use the useEffect() hook on the documentation for calling an async function.  Is there any other way to do it?